content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -3,17 +3,184 @@ title: 'MongoDB'
 metaTitle: 'MongoDB'
 metaDescription: 'Updates on future support for MongoDB.'
```

Comment:
to be updated maybe?

Reply:
Updated to format used by other connectors